nowcasting_dataset/data_sources/satellite/satellite_data_source.py

```diff
@@ -162,67 +162,47 @@ def get_spatial_region_of_interest(
         max_x_and_y_index_height = x_and_y_index_at_center + half_image_size_pixels_height
 
         # Check whether the requested region of interest steps outside of the available data:
-        suggested_reduction_of_image_size_pixels_width = (
-            max(
-                (-min_x_and_y_index_width.min() if (min_x_and_y_index_width < 0).any() else 0),
-                (min_x_and_y_index_width.x_index_at_center - len(data_array.x_geostationary)),
-            )
-            * 2
+        # Need to know how much to pad the outputs, so can do that here
+        min_width_padding = max(-min_x_and_y_index_width.x_index_at_center, 0)
```

Comment:
`min` and `max` confused me a bit, maybe they could be `left` and `right` padding